.gitignore

```diff
@@ -1,9 +1,6 @@
 # Nix Flake stuff
 result
 
-# Generated by nix-pre-commit-hooks
-/.pre-commit-config.yaml
-
```

Comment:
We need this on nix! I did write some stuff in element about how we could solve this another way.